.drone.yml

```diff
@@ -129,7 +597,7 @@ steps:
     ignore_missing: true
```

Comment:
remove this line

Reply:
removed everywhere, switched to latest too as darrens changes are merged in